builtin/audit/file/backend.go

```diff
@@ -149,15 +162,34 @@ func (b *Backend) LogResponse(
 		resp = cp.(*logical.Response)
 
 		// Hash any sensitive information
+
+		// Cache and restore accessor in the auth
+		var accessor string
+		if !b.hashAccessor && auth != nil && auth.Accessor != "" {
+			accessor = auth.Accessor
+		}
 		if err := audit.Hash(b.salt, auth); err != nil {
 			return err
 		}
+		if !b.hashAccessor && auth != nil && auth.Accessor != "" {
```

Comment:
I think this can be simplified to `if accessor != ""` since it will only be assigned to if the same conditions are met above.